assets/css/header.css

```diff
@@ -186,7 +186,15 @@ a:active {
   }
 }
 
+.navbar.burger {
+  display: none;
+}
+
 @media (max-width: 425px) {
```

Comment:
Yah its great but can you make one more little change ? make this max-width 450px .

Reply:
sure thing